File: Estrutura_Condicional/C++/Ex1036/main.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

int main()
{
    double A, B, C, delta, r1, r2;
    std::cin >> A >> B >> C;

    delta = pow(B, 2) - 4 * A * C;

    if (delta < 0 or A == 0) {
        std::cout << "Impossivel calcular\n";
    } else {
        r1 = (-B + sqrt(delta)) / (2 * A);
        r2 = (-B - sqrt(delta)) / (2 * A);

        std::cout << std::fixed << std::setprecision(5);
        std::cout << "R1 = " << r1 << std::endl;
        std::cout << "R2 = " << r2 << std::endl;
    }

    return 0;
}

File: Estrutura_Condicional/Java/Ex1036/Main.java
package Ex1036;

import java.util.Scanner;

public class Main {

    public void calcular_raizes(double a, double b, double c) {
        double delta = Math.pow(b, 2) - 4 * a * c;

        if (delta < 0 || a == 0.0) {
            System.out.println("Impossivel calcular");
            return;
        } else {
            double r1 = (-b + Math.sqrt(delta)) / (2 * a);
            double r2 = (-b - Math.sqrt(delta)) / (2 * a);

            System.out.printf("R1 = %.5f\n", r1);
            System.out.printf("R2 = %.5f\n", r2);
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double a = sc.nextDouble();
        double b = sc.nextDouble();
        double c = sc.nextDouble();
        sc.close();

        Main m = new Main();
        m.calcular_raizes(a, b, c);
    }
}